webviz_subsurface/plugins/_reservoir_simulation_timeseries.py

```diff
@@ -136,6 +136,20 @@ def __init__(
             if self.plot_options.get("date")
             else None
         )
+        # Check if initially plotted vectors exist in data, raise ValueError if not.
+        missing_vectors = {
+            self.plot_options[vec]
+            for vec in set(self.plot_options.keys()).intersection(
+                {"vector1", "vector2", "vector3"}
+            )
+        }.difference(set(self.smry_cols))
+        if missing_vectors:
+            raise ValueError(
+                "Vector(s) {} to be plotted initially in ReservoirSimulationTimeSeries was/were "
+                "not found. Check spelling and that the vector(s) exist(s) in your data.".format(
+                    ", ".join(missing_vectors)
```

Comment:
Use `f`-string instead of `.format()`?

Reply:
Also, maybe alter the error message a bit? It could e.g. be that the vector asked for is indeed in the ensemble data, however, lost on the vector filtering asked for by the user in the plugin.

Reply:
Originally tried f-string, but didn't seem to handle the `", ".join(missing_vectors)` sequence due to the first `"` finishing the `f`-string. If we want to avoid `.format()`, we could of course convert missing_vectors to a string beforehand

Reply:
On the last comment I agree that the we should mention that the vector could be missing in column_keys if that is defined. Not sure how to write it in simple terms. Also do you think the message should be written in singular, plural or a combo like in the current code? Below an expanded draft in just plural:

"Vectors <listed vectors> to be plotted initially in ReservoirSimulationTimeSeries were not found. Check spelling, that the vectors exist in your data and that they are not missing in a list of non-default column_keys."